allauth/socialaccount/providers/apple/provider.py

```diff
@@ -27,5 +39,8 @@ def extract_email_addresses(self, data):
             )
         return ret
 
+    def get_default_scope(self):
+        return ["name"]
```

Comment:
I'd like to be able to disable this

Reply:
@benhowes I think other provider take the name as default scope. Correct me if I am wrong. Both Facebook and Google take the name. We can keep this configurable using settings as well.

Reply:
Yeah, you're probably right, but with apple, the user is given the a screen where they can edit their name if you select the name scope, which is not as simple as with the others where it's completely transparent. 

I will check with the team, we might be able to leave it in.